### PR TITLE
Per line diff

### PR DESCRIPTION
**Pull request by gares**
Requires  coq/coq#6721

### CODE REVIEW

File: timelog2html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+<ol>
+]])
+for i,data_file in ipairs(data_files) do
+	print('<li style="background-color: '..colors[i]..'">' .. data_file .. "</li>")
+end
+print("</ol>")
+
+all_data = {}
+
+for _, data_file in ipairs(data_files) do
```
Comment (JasonGross) [inline timelog2html:63]: If `ipairs ` is like Python's enumerate, why are you using it here just to throw away the number?

File: timelog2html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+	data[#data+1] = {
+		start = last_end+1; stop = string.len(source); time = 0;
+		text = text; lines = lines+1
+		}
+  end
+all_data[#all_data+1] = data
+end
+
+max = 0; 
+for _, data in ipairs(all_data) do
```
Comment (JasonGross) [inline timelog2html:104]: Again, why the ipairs?

File: two_points_on_the_same_branch.sh [hunk 3]
```diff
@@ -373,35 +373,43 @@ fi
...
+
 # The following variable will be set in the following cycle:
 installable_coq_opam_packages=
 
 for coq_opam_package in $coq_opam_packages; do
     echo "DEBUG: coq_opam_package = $coq_opam_package"
 
-    # perform measurements for the NEW commit (provided by the user)
-    export OPAMROOT="$new_opam_root"
+  for RUNNER in NEW OLD; do
```
Comment (JasonGross) [inline two_points_on_the_same_branch.sh:367]: Indentation seems off here

File: timelog2html (hunk 1 above)
Comment (gares) [reply timelog2html:63]: it grants the order in increasing values of the key, `pairs` does not

File: two_points_on_the_same_branch.sh (hunk 3 above)
Comment (gares) [reply two_points_on_the_same_branch.sh:367]: I did not want to re-indent everything (i've added a loop) so used a "half indentation" here...

**Comment by gares** [comment]
The branch now works without any change to Coq

**Comment by gares** [comment]
If https://ci.inria.fr/coq/view/benchmarking/job/benchmark-2-commits/16/consoleFull shows no error I'll merge. I may need @maximedenes to deploy this on pendulum (since one has to `sudo su jenkins`)

**Pull request merged**